django/db/backends/mysql/features.py

```diff
@@ -305,6 +305,9 @@ def supports_transactions(self):
         """
         return self._mysql_storage_engine != "MyISAM"
 
+    uses_savepoints = property(operator.attrgetter("supports_transactions"))
+    can_release_savepoints = property(operator.attrgetter("supports_transactions"))
```

Comment:
I can't see any usage of `can_release_savepoints`.
Please ignore my comment if it's not related.

Reply:
It's used in `transactions` tests.